### PR TITLE
Getting Started Guide: add instructions for other OSs

### PR DESCRIPTION
**Pull request by gvancuts**
Add instructions on how to build the hypervisor and device model
on other operating systems such as Ubuntu/Debian, Fedora/Redhat
and CentOS.

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

File: getting_started/index.rst [hunk 1]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
 **********************
 
 If you would like to build ACRN hypervisor and device model from source,
-follow these steps, using your NUC as a development system:
+follow these steps (based on your development system OS).
 
-#. On your Clear Linux system, install the os-clr-on-clr bundle to get
-   the necessary tools.
+Install build tools and dependencies
+++++++++++++++++++++++++++++++++++++
```
Comment (dbkinder) [inline getting_started/index.rst]: Title underline conventions are:  ``#`` for h1, ``*`` for h2, ``=`` for h3, ``-`` for h4

Use ``=======`` here.

File: getting_started/index.rst [hunk 2]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
 
-   .. code-block:: none
+* On a Clear Linux development system, install the ``os-clr-on-clr`` bundle to get
+   the necessary tools:
```
Comment (dbkinder) [inline getting_started/index.rst]: indentation is significant for ReST.  Align ``the`` with ``One`` on the previous line (delete a leading space)

File: getting_started/index.rst [hunk 3]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
+
+   .. code-block:: console
```
Comment (dbkinder) [inline getting_started/index.rst]: Same here. Keep the indentation of the ``.. code-block`` directive aligned with ``On`` on the previous line and the contents of the code-block aligned to the ``c`` of ``code-block``

File: getting_started/index.rst [hunk 4]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
+
+      $ sudo swupd bundle-add os-clr-on-clr
```
Comment (dbkinder) [inline getting_started/index.rst]: The previous instructions used a ``#`` for the prompt (indicating we're logged in as root).  Are you assuming we're not logged in as root here for Clear Linux (and hence the sudo command)?  We should be consistent on this page.

File: getting_started/index.rst [hunk 5]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
 **********************
 
 If you would like to build ACRN hypervisor and device model from source,
-follow these steps, using your NUC as a development system:
+follow these steps (based on your development system OS).
```
Comment (dbkinder) [inline getting_started/index.rst]: I'd move the ``(based on your development system OS)`` under the next heading - delete it here.

File: getting_started/index.rst [hunk 6]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
 
-#. On your Clear Linux system, install the os-clr-on-clr bundle to get
-   the necessary tools.
+Install build tools and dependencies
+++++++++++++++++++++++++++++++++++++
 
-   .. code-block:: none
+* On a Clear Linux development system, install the ``os-clr-on-clr`` bundle to get
```
Comment (dbkinder) [inline getting_started/index.rst]: Add before this line:

    ARCN development is supported on popular Linux distributions,
    each with their own way to install development tools:

    * On a Clear Linux ...

File: getting_started/index.rst [hunk 7]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
+   the necessary tools:
+
+   .. code-block:: console
+
+      $ sudo swupd bundle-add os-clr-on-clr
+
+* On a Ubuntu/Debian development system:
+
+   .. code-block:: console
```
Comment (dbkinder) [inline getting_started/index.rst]: align the code-block directive (delete one leading space)

File: getting_started/index.rst [hunk 8]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
+      $ sudo apt install git \
+          gnu-efi \
+          libssl-dev \
+          libpciaccess-dev \
+          uuid-dev
+
+* On a Fedora/Redhat development system:
 
-      # swupd bundle-add os-clr-on-clr
+   .. code-blok:: console
```
Comment (dbkinder) [inline getting_started/index.rst]: ``.. code-block`` (misspelled) and delete a leading space to align

File: getting_started/index.rst [hunk 9]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
+
+      $ sudo dnf install gcc \
+          libuuid-devel \
+          openssl-devel \
+          libpciaccess-devel
+
+* On a CentOS development system:
+
+   .. code-block:: console
```
Comment (dbkinder) [inline getting_started/index.rst]: delete leading space to align

File: getting_started/index.rst [hunk 10]
```diff
@@ -281,32 +281,66 @@ Build ACRN from Source
...
+
+      $ sudo yum install gcc \
+             libuuid-devel \
+             openssl-devel \
+             libpciaccess-devel
+
+Build the hypervisor and device model
++++++++++++++++++++++++++++++++++++++
```
Comment (dbkinder) [inline getting_started/index.rst]: ``==========``

File: getting_started/index.rst (hunk 5 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 1 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 6 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 2 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 3 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 7 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 8 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 9 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 10 above)
Comment (gvancuts) [reply getting_started/index.rst]: Done

File: getting_started/index.rst (hunk 4 above)
Comment (gvancuts) [reply getting_started/index.rst]: It is bad practice to do things as root. This is why I have changed this to the `$` prompt and added `sudo`. I don't think we need to be 100% consistent with the previous paragraphs as these are really different operations (installation vs. building). It's very typical to be root when you install things but not when you build things.

**Pull request merged**